azurelinuxagent/ga/update.py

```diff
@@ -324,12 +324,7 @@ def run(self):
         sys.exit(0)
 
     def forward_signal(self, signum, frame):
-        # Note:
-        #  - At present, the handler is registered only for SIGTERM.
-        #    However, clean shutdown is both SIGTERM and SIGKILL.
-        #    A SIGKILL handler is not being registered at this time to
-        #    minimize perturbing the code.
-        if signum in (signal.SIGTERM, signal.SIGKILL):
+        if signum == signal.SIGTERM:
```

Comment:
Dh  you want to catch SIGQUIT as well?

Reply:
I thought SIGQUIT was a keyboard signal?